### PR TITLE
Test that resources fetched by cross-origin CSS are not in the timeline

### PR DESCRIPTION
**Pull request by yoavweiss**
Closes https://github.com/w3c/resource-timing/issues/70

This is a long standing missing test. Seems like all browsers are doing the wrong thing here, which leads me to think we may need to change the spec...

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/23620)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/335608651)

**Comment by igrigorik** [comment]
> This is a long standing missing test. Seems like all browsers are doing the wrong thing here, which leads me to think we may need to change the spec...

Mozilla claims to have address this in https://bugzilla.mozilla.org/show_bug.cgi?id=1180145 — is the test still failing for you? On Chrome side, see: https://bugs.chromium.org/p/chromium/issues/detail?id=532399#c32 — it seems like we still don't have agreement on https://github.com/w3c/ServiceWorker/issues/719. 

/cc @annevk @toddreifsteck @rniwa

**Comment by yoavweiss** [comment]
I see this test failing in the latest Firefox nightly.

I agree that if this information is exposed through Service Workers, there's no point in blocking this info from Resource Timing.

**Comment by annevk** [comment]
@jwatt and @wanderview can maybe help out here. I think our position is that we'd rather block it where possible, even if it's inconsistent.

**Comment by yoavweiss** [comment]
I've rebased and updated the test to include an `@import` test.

Safari Tech Preview seems to pass the test.
Firefox nightly still fails because fonts are present in the timeline. There's an [open issue](https://bugzilla.mozilla.org/show_bug.cgi?id=1423602) on that front.

File: resource-timing/no-entries-for-cross-origin-css-fetched.sub.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE HTML>
+<meta charset=utf-8>
+<title>Make sure that resources fetched by cross origin CSS are not in the timeline.</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body>
+<link rel=stylesheet id=cross_origin_style href="//{{domains[www1]}}:{{ports[http][1]}}{{location[path]}}/../resources/nested.css">
```
Comment (cvazac) [inline resource-timing/no-entries-for-cross-origin-css-fetched.sub.html]: can you please add a comment as to what the `href` resolves to here?

Comment (yoavweiss) [reply resource-timing/no-entries-for-cross-origin-css-fetched.sub.html]: added

**Comment by yoavweiss** [comment]
Commented on the flakiness in the [related bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1423602#c5)

**Review by cvazac — APPROVED** [review]

**Pull request merged**